packages/cloudflare/src/index.ts

```diff
@@ -72,6 +71,17 @@ export type Options = {
 	 * for reference on how these file types are exported
 	 */
 	cloudflareModules?: boolean;
+	/**
+	 * Lists all experimental features the adapter supports.
+	 */
+	experimental?: {
+		/**
+		 * Enables support for Cloudflare Workers assets. Defaults to false.
+		 */
+		cloudflare?: {
+			workerAssets?: boolean;
+		};
```

Comment:
What's the for a `cloudflare` field? The experimental option is **for the cloudflare** adapter, so I suppose there's no need for a `cloudflare` field. Is there something I miss?

Reply:
`cloudflare` as platform features and not adapter or astro features?

Reply:
> platform/host feature and not adapter or astro feature

Oh, I see the point now. However, astro feature VS platform feature is an implementation detail, and it shouldn't be passed down to the user. Other adapters mix astro features and platform features options, which works well.